src/app/pages/wallets/my-wallets/wallet-form-modal/wallet-form-modal.component.html

```diff
@@ -4,6 +4,9 @@ <h1 mat-dialog-title>{{ data ? "Update Wallet" : "Create Wallet" }}</h1>
         <mat-label>Wallet Name</mat-label>
         <input matInput [formControl]='form.controls["name"]'>
     </mat-form-field>
+    <div *ngIf="showNameError()" class="error-text">
```

Comment:
Considering we're using Angular Material you should go with this solution https://material.angular.io/components/form-field/overview#error-messages.
Also please remove `.` from the end of the error message.
Remember also about removing obsolete SCSS/TS code.